src/applications/financial-status-report/components/EmploymentHistorySummaryCard.jsx

```diff
@@ -122,18 +122,18 @@ const EmploymentHistorySummaryCard = ({
 
   return (
     (!job && <EmptyMiniSummaryCard content={emptyPrompt} />) || (
-      <div
-        className="vads-u-border--1px vads-u-margin-y--2 vads-u-padding--0"
+      <va-card
```

Comment:
I'm going to make a discovery/cleanup ticket for this one since it's not using the MiniSummaryCard shared component and we don't really understand why